Database/db-fetch.py

```diff
@@ -0,0 +1,25 @@
+import boto3
+import json
+from decimal import Decimal
+
+class DecimalEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, Decimal):
+            return float(obj)  # Convert Decimal to float for JSON encoding
+        return super(DecimalEncoder, self).default(obj)
+
+dynamodb = boto3.resource('dynamodb', region_name='us-east-1')  # Replace with your region
```

Comment:
Do we need to keep comments if the regions have been replaced as per our project needs? (just confirming, not a blocker)